docker_containers/mindmeld_dep_docker/Dockerfile

```diff
@@ -42,7 +43,7 @@ RUN \
     wget https://artifacts.elastic.co/downloads/elasticsearch/$ES_PKG_NAME.tar.gz && \
     tar xvzf $ES_PKG_NAME.tar.gz && \
     rm -f $ES_PKG_NAME.tar.gz && \
-    mv /$ES_PKG_NAME /elasticsearch
+    mv /$ES_FOLDER_NAME /elasticsearch
```

Comment:
We need the change here cause even though the pkg name is elasticsearch-7.8.0-linux-x86_64, the extracted folder is just called elasticsearch-7.8.0.